From: =?utf-8?q?C=C3=A9dric_Boutillier?= <boutil@debian.org>
Date: Sun, 25 Dec 2016 23:53:41 +0100
Subject: disable test in isolation with autopkgtest

---
 spec/parallel_spec.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/spec/parallel_spec.rb b/spec/parallel_spec.rb
index 9108236..eed3d12 100644
--- a/spec/parallel_spec.rb
+++ b/spec/parallel_spec.rb
@@ -409,7 +409,7 @@ describe Parallel do
       skip if ENV['TRAVIS'] # this randomly hangs on travis
       out = `ruby spec/cases/map_isolation.rb`
       out.should == "1\n2\n3\n4\nOK"
-    end
+    end unless ENV['AUTOPKGTEST_TMP']
 
     it 'sets Parallel.worker_number when run with isolation' do
       skip if ENV['TRAVIS'] # this randomly hangs on travis
